code/modules/food_and_drinks/food/foods/side_dishes.dm

```diff
@@ -107,6 +107,15 @@
 	list_reagents = list("nutriment" = 5, "vitamin" = 1)
 	tastes = list("rice" = 1)
 
+/obj/item/food/snacks/boiledrice/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/stack/seaweed))
+		var/obj/item/stack/seaweed/S = I
+		var/obj/item/food/snacks/onigiri/O = new(get_turf(src))
+		S.use(1)
+		reagents.trans_to(O, reagents.total_volume)
+		qdel(src)
+		return
+	return ..()
```

Comment:
Think you could move the onigiri to the hand, when possible, after the boiledrice is deleted? To avoid food going straight to the floor.